microsoft/testsuites/ltp/ltp.py

```diff
@@ -71,10 +73,13 @@ def dependencies(self) -> List[Type[Tool]]:
     def can_install(self) -> bool:
         return True
 
-    def __init__(self, node: Node, *args: Any, **kwargs: Any) -> None:
+    def __init__(
+        self, node: Node, source_file_path: str, *args: Any, **kwargs: Any
+    ) -> None:
         super().__init__(node, args, kwargs)
         git_tag = kwargs.get("git_tag", "")
         self._git_tag = git_tag if git_tag else self.DEFAULT_LTP_TESTS_GIT_TAG
+        self._tar_file_path = source_file_path
```

Comment:
The variable should be similar, not lost it's meaning, like `self._source_file_path`.

Reply:
I does not quite understand your 1st commet. The "source_file_path" will be parsed in from outside "verify_ltp_lite" in ltpsuite.py file. There should be a var "self._source_file" to take this value in init() and then doing the condition when _install()